Apps/Admin/Common/Models/UserFeedbackView.cs

```diff
@@ -83,9 +84,15 @@ public UserFeedbackView(IList<UserFeedbackTagView> tags)
     public string Email { get; set; } = string.Empty;
 
     /// <summary>
-    /// Gets the feedback admin tags.
+    /// Gets or sets a value indicating whether the feedback has been modified or not.
     /// </summary>
-    public IList<UserFeedbackTagView> Tags { get; } = new List<UserFeedbackTagView>();
+    public bool IsDirty { get; set; }
+
+    /// <summary>
+    /// Gets or sets the feedback admin tags.
+    /// </summary>
+    [SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "Team decision")]
+    public IList<UserFeedbackTagView> Tags { get; set; } = new List<UserFeedbackTagView>();
```

Comment:
See Admin.Common/Models/DelegationInfo.cs for our current approach that avoids the warning and suppression, along with the boilerplate needed for the constructors.